### PR TITLE
Update configuration example for app-id v2 (SSR 5.0) for TLS

### PR DESCRIPTION
**Pull request by robert-patterson**
- in 5.0 and later, TLS no longer creates FIB entries, so hierarhical services and domain-name should be configured

### CODE REVIEW

File: docs/concepts_appid.md [hunk 1]
```diff
@@ -164,6 +164,43 @@ The `example` service needs to have its own `service-route` in order for traffic
 
 With the TLS-based application identification technique, the `application-name` can include a wildcard such as `*.example.com`, which is not possible with the DNS-based approach. This gives administrators a bit more flexibility in defining which traffic to match to services when parsing the X.509 certificates.
 
+#### AppID V2 based on TLS
+
+With App-Id v2 (SSR 5.0 and later), a FIB entry is no longer installed as DPI learns domain-name and IP address from TLS. Instead, heirarchical services and `domain-name` is used.
```
Comment (MichaelBaj) [inline docs/concepts_appid.md:169]: "App ID v2" is not how we've ever marketed this feature. I suggest rewording this to:

"Starting with versions SSR version 5.0, a FIB entry is no longer installed as part of DPI and instead learns domain-names and IP addresses from TLS and maps to hierarchical services using the `domain-name` field.

**Review by MichaelBaj — DISMISSED** [review]

**Review by avinashbhat80 — DISMISSED** [review]

File: docs/concepts_appid.md [hunk 2]
```diff
@@ -164,6 +164,43 @@ The `example` service needs to have its own `service-route` in order for traffic
 
 With the TLS-based application identification technique, the `application-name` can include a wildcard such as `*.example.com`, which is not possible with the DNS-based approach. This gives administrators a bit more flexibility in defining which traffic to match to services when parsing the X.509 certificates.
 
+#### Update for SSR version 5.0
+
+Starting with versions SSR version 5.0, a FIB entry is no longer installed as part of DPI and instead learns domain-names and IP addresses from TLS and maps to hierarchical services using the `domain-name` field.
```
Comment (tjbresnahan) [inline docs/concepts_appid.md:169]: Superfluous `versions` here
```
Starting with SSR version 5.0, ...
```

File: docs/concepts_appid.md [hunk 3]
```diff
@@ -164,6 +164,43 @@ The `example` service needs to have its own `service-route` in order for traffic
...
+
+```
+config
+
+    authority
+
+        service  internet
+            name                  internet
+            description           "www.example.com website"
```
Comment (tjbresnahan) [inline docs/concepts_appid.md:178]: This should be something else, like `Catch-All Internet` or something

File: docs/concepts_appid.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/concepts_appid.md:169]: Starting with SSR version 5.0, a FIB entry is no longer installed as part of DPI. Instead, domain-names and IP addresses are learned from TLS and mapped to hierarchical services using the `domain-name` field.

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]
